test/e2e/View.test.ts

```diff
@@ -0,0 +1,131 @@
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Event_1ujpyos_di" bpmnElement="Event_1ujpyos">
+        <omgdc:Bounds x="722" y="102" width="36" height="36" />
+        <bpmndi:BPMNLabel>
+          <omgdc:Bounds x="714" y="145" width="55" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Event_0spzhad_di" bpmnElement="Event_0spzhad">
+        <omgdc:Bounds x="722" y="202" width="36" height="36" />
+        <bpmndi:BPMNLabel>
+          <omgdc:Bounds x="705" y="245" width="70" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNEdge id="SequenceFlow_0wnb4ke_di" bpmnElement="SequenceFlow_0wnb4ke">
+        <omgdi:waypoint x="390" y="120" />
+        <omgdi:waypoint x="445" y="120" />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="SequenceFlow_0h21x7r_di" bpmnElement="SequenceFlow_0h21x7r">
+        <omgdi:waypoint x="238" y="120" />
+        <omgdi:waypoint x="290" y="120" />
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Flow_0m3z77d_di" bpmnElement="Flow_0m3z77d">
+        <omgdi:waypoint x="495" y="120" />
+        <omgdi:waypoint x="560" y="120" />
+        <bpmndi:BPMNLabel>
+          <omgdc:Bounds x="519" y="102" width="17" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNEdge id="Flow_1cemls1_di" bpmnElement="Flow_1cemls1">
+        <omgdi:waypoint x="470" y="145" />
+        <omgdi:waypoint x="470" y="220" />
+        <omgdi:waypoint x="722" y="220" />
+        <bpmndi:BPMNLabel>
+          <omgdc:Bounds x="479" y="180" width="13" height="14" />
+        </bpmndi:BPMNLabel>
+      </bpmndi:BPMNEdge>
+      <bpmndi:BPMNShape id="Activity_0vcgzmh_di" bpmnElement="Activity_0vcgzmh">
+        <omgdc:Bounds x="560" y="80" width="100" height="80" />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNEdge id="Flow_1qgzzi1_di" bpmnElement="Flow_1qgzzi1">
+        <omgdi:waypoint x="660" y="120" />
+        <omgdi:waypoint x="722" y="120" />
+      </bpmndi:BPMNEdge>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</definitions>
+`;
+  // endregion
+  let graph: Graph;
+
+  beforeAll(async () => {
+    await page.goto('http://localhost:10001');
+    await page.waitForSelector('#graph');
+    graph = new Graph(window.document.getElementById('graph'));
+  });
+
+  beforeEach(() => {
+    jest.setTimeout(100000);
+  });
+
+  it('should display page title', async () => {
+    await expect(page.title()).resolves.toMatch('BPMN Visu JS');
+  });
+
+  it('should display visualization', async () => {
+    // load BPMN
+    graph.load(xmlContent);
+    // model is OK
+    expect(graph.graph.model.cells.hasOwnProperty('StartEvent_1y45yut')).toBeTruthy();
+    // rendering - not OK - when graph is being initialized the window.document.getElementById('graph') is null
+    // await expect(page.waitForSelector('[data-cell-id="StartEvent_1y45yut"]')).resolves.toBeDefined();
```

Comment:
:question: do we keep this commented code?

Reply:
yes for now, for info